tests/Sniffs/Classes/DisallowPhpUnitTestSniffTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Worksome\WorksomeSniff\Tests\Sniffs\Classes;
+
+use SlevomatCodingStandard\Sniffs\TestCase;
+use Worksome\WorksomeSniff\Sniffs\Classes\DisallowPhpUnitTestSniff;
+use Worksome\WorksomeSniff\Sniffs\Classes\ExceptionSuffixSniff;
+
+class DisallowPhpUnitTestSniffTest extends TestCase
+{
+    public function testNoErrors(): void
+    {
+        $report = self::checkFile(__DIR__ . '/../../Resources/Sniffs/Classes/DisallowPhpUnitTestSniff/PestTest.php.stub');
+
+        self::assertNoSniffErrorInFile($report);
+    }
+
+    public function testErrorWhenPhpUnitTestAnnotationEncountered(): void
```

Comment:
Consider adding a test that also checks when multiple annotations is on the method and one where someone added test annotation to a non method 😅

Reply:
Added